### PR TITLE
[Oil-251] add webinar form scaffolding

### PR DESCRIPTION
**Pull request by phogel**
- prepare logic
- needs another update when backend api is finished

### CODE REVIEW

File: src/layout/index/webinar-form/webinar-form.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    event.preventDefault();
+  }
+
+  submitComplete() {
+    this.setState({
+      submitted: true
+    });
+  }
+
+  handleChange(event) {
```
Comment (subsend) [inline src/layout/index/webinar-form/webinar-form.js]: Do we need the transfer values ​​with each click or it is only enough to send the values on form submit?

File: src/layout/index/webinar-form/webinar-form.scss [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      background-color: $lighter-link-color;
+    }
+  }
+
+  .webinar__success-message {
+    font-weight: 400;
+    font-style: italic;
+  }
+
+  @media only screen and (max-width: $breakpoint-desktop) {
```
Comment (subsend) [inline src/layout/index/webinar-form/webinar-form.scss]: maybe nesting into associated classes?

File: src/layout/index/webinar-form/webinar-form.scss [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+      width: 19%;
+      margin: 1vh;
+    }
+
+    form label {
+      width: 100%;
+      clear: both;
+    }
+
+    form input[type="text"] {
```
Comment (subsend) [inline src/layout/index/webinar-form/webinar-form.scss]: what about type="email"?

File: src/layout/index/webinar-form/webinar-form.scss [hunk 4]
```diff
@@ -0,0 +1,97 @@
+@import '../../variables';
+
+.support .webinar {
```
Comment (subsend) [inline src/layout/index/webinar-form/webinar-form.scss]: do we need so deep selector ".support .webinar"? maybe just .webinar?

File: src/layout/index/webinar-form/webinar-form.js (hunk 1 above)
Comment (phogel) [reply src/layout/index/webinar-form/webinar-form.js]: The `handleChange()` makes sure that the state values correspond to the ones the user has put in the input fields. Only on form submission `handleSubmit` is called.

**Pull request merged**